deploy/kubernetes/elastic-agent-standalone-kubernetes.yaml

```diff
@@ -452,38 +467,42 @@ data:
           - data_stream:
               dataset: system.process
               type: metrics
-            process.include_top_n.by_memory: 5
             period: 10s
             processes:
               - .*
             process.include_top_n.by_cpu: 5
-            process.cgroups.enabled: false
+            process.include_top_n.by_memory: 5
             process.cmdline.cache.enabled: true
+            process.cgroups.enabled: false
+            process.include_cpu_ticks: false
             metricsets:
               - process
             process.include_cpu_ticks: false
-            system.hostfs: /hostfs
```

Comment:
@elastic/elastic-agent-data-plane we also removed this. Is not needed in latest 1.20.4 version right?

Reply:
There were a collection of changes that made `system.hostfs: /hostfs` unnecessary. @fearful-symmetry can confirm.